dev/benchmarks/macrobenchmarks/lib/src/web/recorder.dart

```diff
@@ -511,20 +578,24 @@ double _computeMean(Iterable<double> values) {
 /// See also:
 ///
 /// * https://en.wikipedia.org/wiki/Standard_deviation
-double _computeStandardDeviationForPopulation(Iterable<double> population) {
+double _computeStandardDeviationForPopulation(Iterable<num> population) {
   final double mean = _computeMean(population);
   final double sumOfSquaredDeltas = population.fold<double>(
     0.0,
-    (double previous, double value) => previous += math.pow(value - mean, 2),
+    (double previous, num value) => previous += math.pow(value - mean, 2),
   );
   return math.sqrt(sumOfSquaredDeltas / population.length);
 }
 
+String _percent(double value) {
```

Comment:
Maybe `_ratioToPercent` (since it also multiplies by 100 rather than just add "%")?